2022-03-04 11:34:06.121   ERROR org.apache.flink.runtime.taskmanager.Task                     - method:org.apache.flink.runtime.taskmanager.Task$TaskCanceler.run(Task.java:1541)
Error while canceling the task Source: Custom Source (1/1)#0.
java.lang.NullPointerException
	at transform.process.state.broadcastState.MySQLSource.cancel(MySQLSource.java:51)
	at org.apache.flink.streaming.api.operators.StreamSource.cancel(StreamSource.java:160)
	at org.apache.flink.streaming.runtime.tasks.SourceStreamTask.cancelTask(SourceStreamTask.java:204)
	at org.apache.flink.streaming.runtime.tasks.SourceStreamTask.cancelTask(SourceStreamTask.java:185)
	at org.apache.flink.streaming.runtime.tasks.StreamTask.cancel(StreamTask.java:717)
	at org.apache.flink.runtime.taskmanager.Task$TaskCanceler.run(Task.java:1538)
	at java.lang.Thread.run(Thread.java:748)
